runbotics/runbotics-orchestrator-ui/src/views/process/ProcessBuildView/Modeler/ConfigureActionPanel/widgets/ElementAwareAutocompleteWidget.tsx

```diff
@@ -145,13 +157,42 @@ const ElementAwareAutocompleteWidget: FC<WidgetProps> = (props) => {
                         element.businessObject.actionId === 'variables.assignList',
                 );
 
-            const variables = assignVariablesElements
+            const globalVariables = assignGlobalVariablesElements
+                .map((assignGlobalVariablesElement) => {
+                    const inputOutput: CamundaInputOutputElement = assignGlobalVariablesElement.businessObject
+                        ?.extensionElements?.values[0] as CamundaInputOutputElement;
+                    if (inputOutput) {
+                        const [globalVariable] = inputOutput.inputParameters.filter(
+                            (inputParameter) => 
+                                inputParameter.name === 'globalVariable',
+                        );
+
+                        const globals = store.getState().globalVariable.globalVariables;
```

Comment:
don't use redux like that, we have a dedicated slice for it. Use `useSelector` hook to get it, see how it is implemented in e.g. global variables main page